src/blocks/remote-data-container/components/placeholders/PlaceholderError.scss

```diff
@@ -0,0 +1,63 @@
+// selected error placeholder
+.rdb-container.is-selected .components-placeholder.remote-data-container-error {
+
+	.components-placeholder__label > svg {
+		fill: var(--Gutenberg-Alert-Red, #cc1818);
+	}
+
+	.remote-data-container-error__content {
+		background: var(--Gutenberg-Gray-100, #f0f0f0);
+		border-radius: 2px;
+		color: var(--Gutenberg-Gray-900, #1e1e1e);
+		line-height: 1.5;
+		padding: 12px 16px;
+		white-space: pre-wrap;
+		width: auto;
+	}
+
+	// hide overlay icon
+	.remote-data-container-error__icon-overlay {
+		display: none;
+	}
+}
+
+// unselected error placeholder
+.rdb-container:not(.is-selected) .components-placeholder.remote-data-container-error {
```

Comment:
I wonder if it would be better to at least some have test or UI showing when it's not selected?

Reply:
Not sure I follow. Do you mean a test to ensure that the selected state is applied? That is handled by core so I'm not sure how we would unit test it. Or do you mean some other UI state should be applied?

Reply:
I meant to say text, that was a typo apologies. I've corrected it to say text, so that one would be able to tell that the block is in an error state which is why it appears that way. It won't render when the post is viewed, so this would be just for the block editor

Reply:
We are closely following @jarekmorawski's design, so I'll defer to him on this suggestion. It does bring up a general a11y concern to make sure that the error is discoverable / announceable.